addons/opsportal/1.0.x/opsportal-1.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-1"
     appversion.kubeaddons.mesosphere.io/opsportal: "1.0.0"
     endpoint.kubeaddons.mesosphere.io/opsportal: /ops/portal/
-    values.chart.helm.kubeaddons.mesosphere.io/opsportal: "https://raw.githubusercontent.com/mesosphere/charts/4155f480571eaf82c64ddd63d3d334b1105d0591/stable/opsportal/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/opsportal: "https://raw.githubusercontent.com/mesosphere/charts/master/stable/opsportal/values.yaml"
```

Comment:
seems like this line was referring to a specific git hash instead of master (like kommander / other addons). I'd assume this is a bug? cc @shaneutt @alejandroEsc

Reply:
Technically not a bug, it's a hack. This is intentionally set to a specific sha which points to the same version as the included application, such as to make sure the link is valid as per that version.

Reply:
ok, so what are we supposed to do? update to new sha once https://github.com/mesosphere/charts/pull/316 is merged? sorry for these questions. its so much new stuff to understand 🙈

Reply:
No worries! yes. once the chart merges then link with the SHA that matches the chart version. :+1:

Reply:
ok, thanks for clarification 🙇

Reply:
Hm a little lost as to why we track the chart version in two places? The actual chart version ref and then this link as well? 🤔

Reply:
removed that part from the PR, I still fully understand whats happening and it seems like it worked before.